backend/app/api/endpoints/projects.py

```
@@ -0,0 +1,241 @@
+# ============================================================================
+
+
+@router.post(
+    "/{project_id}/tasks",
+    response_model=AddTaskToProjectResponse,
+    status_code=status.HTTP_201_CREATED,
+)
+def add_task_to_project_endpoint(
+    project_id: int = Path(..., description="Project ID"),
+    task_data: ProjectTaskCreate = None,
+    current_user: User = Depends(get_current_user),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)